.changeset/kind-humans-explode.md

```diff
@@ -0,0 +1,6 @@
+---
+"@salt-ds/data-grid": patch
+---
+
+bug - error due to scrollTop not resetting on each pagination causing start > end when passed into NumberValue
+fix - guard to return an empty NumberValue when start > end preventing an error when scrollTop doesn't reset
```

Comment:
something like this is best for consumers

Reply:
does a changeset not also help to provide visibility to developers without them needing to dig into the code ?

Reply:
hmm, I think the general user wouldnt care how it was fixed just that it is fixed, and if they do then would probably dig into it

Reply:
The changeset is meant to explain to a consumer what has changed and how it would impact them. For this change I would probably stick with:

```suggestion
Fixed the Data Grid crashing when a user scrolls to the bottom and navigates to the last page
```